build/process-template.sh

```diff
@@ -16,6 +16,8 @@
 ################################################################################
 set -e
 
+sudo apt-get install -y gettext-base
```

Comment:
what is this a requirement for?

Reply:
it is needed for the "envsubst " command used later in the same script, it was failing on ppc64le without that

Reply:
ah ok